docs/reference/supported-environments.md

```diff
@@ -32,14 +32,14 @@ We highly recommend running Camunda Platform 8 Self-Managed in a Kubernetes envi
 
 Requirements for the components can be seen below:
 
-| Component   | Java version | Other requirements                                                                                                                            |
-| ----------- | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------- |
-| Zeebe       | OpenJDK 17+  | Elasticsearch 7.16.x, 7.17.x (only if Elastic exporter is used), 8.5.x, 8.6.x, OpenSearch 2.5.x (only if OpenSearch exporter is used)         |
-| Operate     | OpenJDK 17+  | Elasticsearch 7.16.x, 7.17.x, 8.5.x, 8.6.x                                                                                                    |
-| Tasklist    | OpenJDK 17+  | Elasticsearch 7.16.x, 7.17.x, 8.5.x, 8.6.x                                                                                                    |
-| Identity    | OpenJDK 17+  | Keycloak 21.x<br/>PostgreSQL 14.x, 15.x                                                                                                       |
-| Optimize    | OpenJDK 11+  | Elasticsearch 7.16.x, 7.17.x, 8.5.x, 8.6.x                                                                                                    |
-| Web Modeler | -            | Keycloak 21.x<br/>PostgreSQL 13.x, 14.x, 15.x, Amazon Aurora PostgreSQL 13.x, 14.x, 15.x (other database systems are currently not supported) |
+| Component   | Java version | Other requirements                                                                                                                                        |
+| ----------- | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Zeebe       | OpenJDK 17+  | Elasticsearch 8.5.x+, Amazon OpenSearch 2.5.x+ (only if [OpenSearch exporter](../self-managed/zeebe-deployment/exporters/opensearch-exporter.md) is used) |
+| Operate     | OpenJDK 17+  | Elasticsearch 8.5.x+, Amazon OpenSearch 2.5.x+                                                                                                            |
+| Tasklist    | OpenJDK 17+  | Elasticsearch 8.5.x+, Amazon OpenSearch 2.5.x+                                                                                                            |
+| Identity    | OpenJDK 17+  | Keycloak 21.x<br/>PostgreSQL 14.x, 15.x                                                                                                                   |
+| Optimize    | OpenJDK 11+  | Elasticsearch 8.5.x+, Amazon OpenSearch 2.5.x+                                                                                                            |
```

Comment:
Shouldn't this be ES 8.7? We are moving to 8.7 with [this epic](https://github.com/camunda/product-hub/issues/1117), and one of the reasons is because <8.7 will be out of support. I think that generally we don't support environments that are no longer supported by the provider

Reply:
Ah sure. Typo! Thanks

Reply:
Corrected. Requested re-review @RomanJRW